### ISSUE TITLE
Confusing message about Closing Mariadb

### ISSUE THREAD

**Issue by tosiara** [Motion-Project/motion#1284]
1. version [x.y.z, hash, other]: https://github.com/Motion-Project/motion/commit/02fed650b3281c430078251ce8a3225c3dc52310

Motion compiled with both mariadb and sqlite support.
The config is using sqlite.
On exit there is a confusing message about `Closing Mariadb`:

```
[0:motion] [NTC] [ALL] main: Threads finished
[0:motion] [NTC] [ALL] main: Motion terminating
[0:motion] [DBG] [ALL] dbse_global_deinit: Closing Mariadb
[0:motion] [DBG] [ALL] dbse_global_deinit: Database closed
```
Config used:

```
database_type sqlite3
database_dbname /home/user/motion.db
sql_log_picture on
sql_query insert into events(camera, event, time, type, frame, filename) values('%t', '%v.%C', '%Y-%m-%d %T', '%n', %q, '%f')
```
The message seems to be harmless, but still confusing

**Comment by wbclay** [comment]
I have corrective code ready to push when permission to do so is granted

**Comment by tosiara** [comment]
You are welcome to submit pull request for review

**Comment by Mr-Dave** [comment]
closed via #1310
